test/e2e_node/podresources_test.go

```diff
@@ -702,6 +702,26 @@ var _ = SIGDescribe("POD Resources [Serial] [Feature:PodResources][NodeFeature:P
 					podresourcesListTests(ctx, f, cli, nil)
 					podresourcesGetAllocatableResourcesTests(ctx, cli, nil, onlineCPUs, reservedSystemCPUs)
 				})
+				ginkgo.It("should account for resources of pods in terminal phase", func(ctx context.Context) {
+					pd := podDesc{
+						cntName:    "e2e-test-cnt",
+						podName:    "e2e-test-pod",
+						cpuRequest: 1000,
+					}
+					pod := makePodResourcesTestPod(pd)
+					pod.Spec.Containers[0].Command = []string{"sh", "-c", "sleep 10s"}
```

Comment:
minor(ish): can we sleep for a shorter amount, like 1s, saving wall clock time? Or maybe don't sleep at all, use `/bin/true` as entry point. After all, the only thing that matters is the pod goes `Succeeeded`, right?

Reply:
`/bin/true` it is. Thanks!